### DISCUSSION TITLE
How to get data to the power card

### DISCUSSION THREAD

**Discussion by ibby24** [ulic75/power-flow-card#151] [Q&A]
Hello,

Completely new to home assistant and power flow card. 

What I have done. 

Installed HACOS, and the power card.
Created a new dashboard and added the plugin. 

Saved it, rebooted the home assistant but it has no data, how do I get it to go away and take my data and show me on the power flow card ?

Thanks

**Comment by kpfleming** [comment]
Have you read the setup instructions [here](https://github.com/ulic75/power-flow-card#using-the-card)?
